realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -169,6 +169,30 @@ private boolean containsDot(String s) {
         }
     }
 
+    // isNull and isNotNull
+
+    /**
+     * Add condition "Is relationship null?"
```

Comment:
A bit convoluted. Perhaps "Test if a field is null. Only works for relationships and RealmLists"